report/verification/mes/tmap_1c.md

```diff
@@ -0,0 +1,128 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# TMAP7 V&V Val-1c
+
+```{tags} 1D, MES, transient
+```
+
+This verification case {cite}`ambrosek_verification_2008` from TMAP7's V&V document consists of a semi-infinite slab with no traps under a constant concentration $C_0$ on the first $10 m$ of the slab.
+
++++
+
+## FESTIM Code
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+import festim as F
+import numpy as np
+import sympy as sp
+from matplotlib import pyplot as plt
+
+preloaded_length = 10 # m
+C_0 = 1 # atom m^-3
+D = 1 # 1 m^2 s^-1
+exact_solution = C_0/2 * (
+    2*sp.erf(F.x / sp.sqrt(4*D*F.t)) 
+    - sp.erf((F.x - preloaded_length) / sp.sqrt(4*D*F.t))
+    - sp.erf((F.x + preloaded_length) / sp.sqrt(4*D*F.t))
+)
+
+model = F.Simulation()
+
+### Mesh Settings ###
+vertices = np.concatenate([
+    np.linspace(0, 10, 200),
+    np.linspace(10, 100, 300),
+])
+
+model.mesh = F.MeshFromVertices(vertices)
+
+model.boundary_conditions = [
+    F.DirichletBC(surfaces=[1], value=0, field="solute")
+]
+
+initial_concentration = sp.Piecewise(
+    (C_0, F.x <= preloaded_length),
+    (0, True)
+)
+model.initial_conditions = [
+    F.InitialCondition(field="solute", value=initial_concentration)
+]
+
+model.materials = [F.Material(id=1, D_0=D, E_D=0)]
+
+model.T = F.Temperature(500)  # ignored in this problem
+
+model.dt = F.Stepsize(
+    initial_value=0.01,
+    stepsize_change_ratio=1.1,
+)
+
+test_points = [0.5, preloaded_length, 12] #m
+final_times = [100, 100, 50]
+derived_quantities = F.DerivedQuantities(
+    [F.PointValue("solute", x=v) for v in test_points]
+)
+model.exports = [derived_quantities]
+
+model.settings = F.Settings(
+    absolute_tolerance=1e-10,
+    relative_tolerance=1e-10,
+    final_time=max(final_times)
+)
+
+model.initialise()
+model.run()
+```
+
+## Comparison with exact solution
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+fig, axs = plt.subplots(1, len(test_points))
```

Comment:
For comparison, if you have a row, please use the same y axis. 
```
fig, axs = plt.subplots(1, len(test_points), sharey=True)
```

Otherwise, if you want to zoom on the data, ok but have a column instead:
```
fig, axs = plt.subplots(len(test_points), 1, sharex=True)
```